src/middleware/sessionStore.js

```diff
@@ -0,0 +1,56 @@
+import {connectionAPI, connectionDefault} from '../config'
+import {Schema} from 'mongoose'
+
+/**
+ * Session Store
+ *
+ * To be able to store the session in a Mongo collection instead of just saving it in memory
+ * 
+ */
+
+const SessionSchema = new Schema({
+  _id: String,
+  data: Object,
```

Comment:
Is there a possiblity with mongo to have a ref to the user object? Rather than having a json that contains the user id ?Or is it the way passport wants it ?

Reply:
I don't think I have the possibility to change that
According to the code passport is accessing a key first and then user:

![image](https://user-images.githubusercontent.com/48720958/216094559-34d7a8d1-3fa7-4900-91e3-85cd603113d8.png)


data should be : { passport : { user : ... } } with passport is the default key